packages/amplify-e2e-tests/Readme.md

```diff
@@ -4,27 +4,27 @@ This packages contains end to end tests that are run in CircleCI to ensure that
 
 ## Setup
 
-To run the tests locally, you need to have your AWS credentials stored in a `.env` file of this package. These values are used to configure the test projects.
+To run the tests locally, you need to have your personal AWS credentials stored in a `.env` file of this package. These values are used to configure the test projects.
 
-Please see sample.env for the keys that are expected in your `.env` file.
+Please see sample.env for the keys that are expected in your `.env` file. Or, you can copy "Copy bash/zsh" directly from Isengard of you personal account, and paste it in the terminal.
```

Comment:
I know it's probably not meant for external contributors to run e2e tests, we should still not expose internal concepts such as `Isengard` in our readmes

Reply:
Make sense. Updated.